browser/ui/views/tabs/brave_tab_strip.cc

```diff
@@ -326,6 +326,11 @@ void BraveTabStrip::UpdateTabContainer() {
   }
 
   hover_card_controller_->SetIsVerticalTabs(using_vertical_tabs);
+
+  if (const auto active_index = GetActiveIndex(); active_index) {
+    // In order to update shadow state, call ActiveStateChanged().
+    tab_at(active_index.value())->ActiveStateChanged();
```

Comment:
Curious why we need to call this here? Is there a specific scenario that needs this call?
I think `BraveTab::ActiveStateChanged()` will be called when tab's active state changed?

Reply:
got it.